javaparser-core/src/main/java/com/github/javaparser/printer/PrettyPrinterConfiguration.java

```diff
@@ -150,4 +151,13 @@ public PrettyPrinterConfiguration setOrderImports(boolean orderImports) {
         this.orderImports = orderImports;
         return this;
     }
+
+    public int getMaxEnumConstantsToAlignHorizontally() {
+        return maxEnumConstantsToAlignHorizontally;
+    }
+
+    public PrettyPrinterConfiguration setMaxEnumConstantsToAlignHorizontally(int maxEnumConstantsToAlignHorizontally) {
```

Comment:
Do we have a number to indicate infinite? Like -1

Reply:
Oh, could use some Javadoc, I guess.